pkg/runutil/runutil.go

```diff
@@ -158,3 +159,41 @@ func ExhaustCloseWithErrCapture(err *error, r io.ReadCloser, format string, a ..
 
 	*err = merr.Err()
 }
+
+// DeleteAllExceptDirs deletes all files and directories inside the given
+// dir except for the ignoreDirs directories.
+func DeleteAllExceptDirs(dir string, ignoreDirs []string) error {
```

Comment:
```suggestion
func DeleteAll(dir string, ignoreDirs ..string) error {
```

This would be clear enough IMO WDYT?

Reply:
yep